gocd/api/stage.py

```diff
@@ -0,0 +1,71 @@
+from gocd.api.endpoint import Endpoint
+
+__all__ = ['Stage']
+
+
+class Stage(Endpoint):
+    base_path = 'go/api/stages/{id}'
+
+    def __init__(self, server, pipeline_name, pipeline_counter, stage_name):
```

Comment:
After checking the API, it seems the only time you need the pipeline counter, you also need the stage counter. So this seems like a good idea in the end.

I didn't notice that, because I was only interested in the instance() method.